medical-portal/src/API/RSBC.DMF.MedicalPortal.API/Controllers/DocumentController.cs

```diff
@@ -175,5 +175,63 @@ public async Task<IActionResult> UploadDriverDocument([FromForm] IFormFile file,
         }
 
 
+        [HttpPost("claimDmer")]
+        [ProducesResponseType(typeof(CaseDocument), 200)]
+        [ProducesResponseType(401)]
+        [ProducesResponseType(500)]
+        public async Task<IActionResult> UpdateClaimDmerOnDocument([FromRoute] string documentId)
```

Comment:
both of these services need `[Authorize(Policy = Policies.MedicalPractitioner)]`

Reply:
Added Authorization to both end points